simulation.go

```diff
@@ -325,6 +325,12 @@ func (s *simscreen) DisablePaste() {
 	s.paste = false
 }
 
+func (s *simscreen) EnableFocus() {
+}
+
+func (s *simscreen) DisableFocus() {
+}
+
```

Comment:
I didn't bother setting a flag on `s` here like it's done for mouse and paste; as far as I can tell, `s.mouse` and `s.paste` are not used anywhere, so I don't understand why they exist. Let me know if I'm missing something, or whether you'd like me to maintain an unused `s.focus` flag for consistency.